docs/public-networks/reference/cli/options.md

```diff
@@ -2787,6 +2787,43 @@ A list of comma-separated TLS protocols to support. The default is `DEFAULT_TLS_
     The singular `--rpc-http-tls-protocol` and plural `--rpc-http-tls-protocols` are available and are two names for
     the same option.
 
+### `rpc-max-logs-range`
+
+=== "Syntax"
+
+    ```bash
+    --rpc-max-logs-range=<INTEGER>
+    ```
+
+=== "Example"
+
+    ```bash
+    --rpc-max-logs-range=500
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    BESU_RPC_MAX_LOGS_RANGE=500
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    rpc-max-logs-range=500
+    ```
+
+When using [`eth_getLogs`](../api/index.md#eth_getlogs), the maximum number of blocks to get logs
+from.
+Set to 0 to specify no limit.
+The default is 1000.
+
+!!! warning
+
+    Using `eth_getLogs` to get logs from a large range of blocks, especially an entire chain from
+    its genesis block, can cause Besu to hang and never return a response.
```

Comment:
"...genesis block, _**may cause Besu to hang for an indeterminable amount of time while the response is generated.**_"

Does that sound more precise?